Buildings/Experimental/DHC/CentralPlants/Cooling/Controls/ChilledWaterPumpSpeed.mo

```diff
@@ -0,0 +1,151 @@
+within Buildings.Experimental.DHC.CentralPlants.Cooling.Controls;
+model ChilledWaterPumpSpeed
+  "Controller for up to two headed variable speed chilled water pumps"
```

Comment:
```suggestion
  "Controller for up to two variable speed chilled water pumps"
```
(I might be wrong here. I just don't know what headed refers to in this context)

Reply:
This came from an earlier discussion here: https://github.com/lbl-srg/modelica-buildings/pull/2078#discussion_r497861450. However, we've removed the "headed" to avoid confusion.